### DISCUSSION TITLE
Why are { } scraped out from string when they are passed into node ?

### DISCUSSION THREAD

**Discussion by Infinitusvoid** [Comfy-Org/ComfyUI#3258] [General]
When I pass into a node the :

-------------------------
#version 430

layout (local_size_x = 16, local_size_y = 16) in;
layout (binding = 0, rgba32f) uniform image2D img_output;
layout(std140, binding = 0) uniform UniformBlock {
	float uniform_data;
} ubo;

void main()
{
	ivec2 pixel_coords = ivec2(gl_GlobalInvocationID.xy);
	vec4 color = vec4(pixel_coords.x / 800.0, pixel_coords.y / 600.0, 0.0, 1.0);
	color.g = ubo.uniform_data; // Example of using the uniform data
	imageStore(img_output, pixel_coords, color);
}
-------------------------

What I get in python code is :

-------------------------
#version 430

layout (local_size_x = 16, local_size_y = 16) in;
layout (binding = 0, rgba32f) uniform image2D img_output;
layout(std140, binding = 0) uniform UniformBlock
        float uniform_data;
 ubo;

void main()

        ivec2 pixel_coords = ivec2(gl_GlobalInvocationID.xy);
        vec4 color = vec4(pixel_coords.x / 800.0, pixel_coords.y / 600.0, 0.0, 1.0);
        color.g = ubo.uniform_data;
        imageStore(img_output, pixel_coords, color);
-------------------------

screen shoots for example 

<img width="252" alt="node_input" src="https://github.com/comfyanonymous/ComfyUI/assets/84730862/5d42efe8-be50-4204-b688-1f04b7aec1d8">
<img width="477" alt="node_output" src="https://github.com/comfyanonymous/ComfyUI/assets/84730862/38537107-e9e6-4a4d-979d-d4a1aaf4a6e2">

**Comment by ltdrdata** [comment]
Is it your own node?

Then turn off the `dynamicPrompts` 
```
"text": ("STRING", {"multiline": True, "dynamicPrompts": False}),
```

**Reply by Infinitusvoid** [reply]
Thanks that solves it
